lib/mirage.mli

```diff
@@ -53,6 +53,9 @@ val qubesdb: qubesdb typ
 val default_qubesdb: qubesdb impl
 (** A default qubes database, guessed from the usual valid configurations. *)
 
+type qrexec
```

Comment:
Some doc-comments are needed here. Most people reading this aren't going to know what qrexec (or Qubes) is.